app/helpers.php

```diff
@@ -76,6 +77,26 @@ function fragment_slug($name, array $replacements = []): string
     return str_slug($translation);
 }
 
+function fragment_image($name, $conversion = 'thumb'): string
+{
+    if (! str_contains($name, '.')) {
+        return $name;
+    }
+
+    [$group, $key] = explode('.', $name, 2);
+
+    $fragment = Fragment::with('media')
+        ->where('group', $group)
+        ->where('key', $key)
+        ->first();
+
+    if (! $fragment) {
+        return $name;
```

Comment:
When this would fail, I think it's best to see the name of the fragment for which this failed instead of an empty string.